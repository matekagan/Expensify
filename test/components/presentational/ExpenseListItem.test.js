import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../../src/components/presentational/ExpenseListItem';
import testExpenses from '../../fixtures/expenses';

test('Should render expense list item with data', () => {
    const expense = testExpenses[0];
    const wrapper = shallow(<ExpenseListItem key={expense.id} {...expense} />);
    expect(wrapper).toMatchSnapshot();
});
