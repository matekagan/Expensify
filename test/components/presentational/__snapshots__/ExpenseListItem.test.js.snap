// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expense list item with data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    $2.00
    -
    January 31st, 1970
  </p>
</div>
`;
